src/interaction/InteractionManager.js

```diff
@@ -788,19 +788,19 @@ export default class InteractionManager extends EventEmitter
             interactiveParent = false;
         }
 
-        // it has a mask! Then lets hit test that before continuing..
-        if (hitTest && displayObject._mask)
+        // it has a filterArea! Same as mask but easier, its a rectangle
+        if (hitTest && displayObject.filterArea)
         {
-            if (!displayObject._mask.containsPoint(point))
+            if (!displayObject.filterArea.contains(point.x, point.y))
             {
                 hitTest = false;
             }
         }
 
-        // it has a filterArea! Same as mask but easier, its a rectangle
-        if (hitTest && displayObject.filterArea)
+        // it has a mask! Then lets hit test that before continuing if no hitArea or filterArea is present
+        if (hitTest && displayObject._mask && !displayObject.hitArea && !displayObject.filterArea)
```

Comment:
Ahh that does! Just gonna throw this out there - but wander if we should even test for filter area. The purpose was to optimise filters rather than interaction. 

The majority of the time the filter area is set to something massive too, offering very little in way of optimisation here..

I think we should remove the check.. 

What are your thoughts?

Cheers!